src/components/BallotProxyMetadata/index.js

```diff
@@ -16,11 +17,12 @@ export class BallotProxyMetadata extends React.Component {
       /*4*/ { value: '4', label: ballotStore.ProxyBallotType[4] }, // VotingToChangeProxy
       /*5*/ { value: '5', label: ballotStore.ProxyBallotType[5] }, // BallotsStorage
       /*6*/ { value: '7', label: ballotStore.ProxyBallotType[7] }, // ValidatorMetadata
-      /*7*/ { value: '8', label: ballotStore.ProxyBallotType[8] } // ProxyStorage
+      /*7*/ { value: '8', label: ballotStore.ProxyBallotType[8] }, // ProxyStorage
+      /*8*/ { value: '9', label: ballotStore.ProxyBallotType[9] } // RewardByBlock
     ]
 
-    if (!contractsStore.proxyStorage || !contractsStore.proxyStorage.doesMethodExist('getValidatorMetadata')) {
-      options.splice(6) // keep 0-5 and remove 6-... items if ProxyStorage is old
+    if (getNetworkName(contractsStore.netId).toLowerCase() != 'sokol') {
```

Comment:
Let's use constant, defined in `/src/utils/constants.js` for 'sokol'. Btw, it used in different places in the code.

Reply:
Fixed in https://github.com/poanetwork/poa-dapps-voting/pull/194/commits/3ef692cfa00a42d5c33db8b2dc22a54ac402cd90.